OWML.Launcher/SocketListener.cs

```diff
@@ -27,9 +27,7 @@ private void SetupSocketListener()
             }
             catch (SocketException ex)
             {
-                Console.ForegroundColor = ConsoleColor.Red;
-                Console.WriteLine("Error in socket listener : " + ex);
-                Console.ForegroundColor = ConsoleColor.Gray;
+                ConsoleUtils.WriteLineWithColor(ConsoleColor.Red, $"Error in socket listener: {ex}");
```

Comment:
Maybe you should have a method in `SocketListener` just to log these errors, like
```
private void WriteToConsole(MessageType type, string message) {
    ConsoleUtils.WriteLineWithColor(ConsoleUtils.ConsoleColorFromMessageType(type), message);
}
```

Reply:
Or maybe you can drop WriteLineWithColor and just have a method that writes by type? Why would you need to manually specify a color?

Reply:
done